pkg/provider-local/controller/backupentry/actuator.go

```diff
@@ -48,7 +49,8 @@ func (a *actuator) GetETCDSecretData(_ context.Context, _ logr.Logger, _ *extens
 }
 
 func (a *actuator) Delete(_ context.Context, log logr.Logger, be *extensionsv1alpha1.BackupEntry) error {
-	path := filepath.Join(a.backBucketPath, be.Spec.BucketName, be.Name)
+	entryName := strings.TrimPrefix(be.Name, "source-")
```

Comment:
There is a constant for this, you should use `v1beta1constants.BackupSourcePrefix` if possible.